game/dota_addons/dota_imba/scripts/vscripts/modifier/modifier_imba_silencer_int_steal.lua

```diff
@@ -0,0 +1,79 @@
+modifier_imba_silencer_int_steal = class({})
+
+function modifier_imba_silencer_int_steal:OnCreated( kv )
+	self.steal_range = kv.steal_range or 925
+	self.steal_amount = kv.steal_amount or 2
+
+	if IsServer() then
+		self.parent = self:GetParent()
+		self:SetStackCount( self:GetStackCount() )
+		self.parent:CalculateStatBonus()
+	end
+end
+
+-- Manta Meta a possibility
+function modifier_imba_silencer_int_steal:AllowIllusionDuplicate()
+	return false
+end
+
+function modifier_imba_silencer_int_steal:RemoveOnDeath()
+	return false
+end
+
+function modifier_imba_silencer_int_steal:GetAttributes()
+	return MODIFIER_ATTRIBUTE_IGNORE_INVULNERABLE + MODIFIER_ATTRIBUTE_PERMANENT
+end
```

Comment:
MODIFIER_ATTRIBUTE_PERMANENT seems to do nothing for lua modifiers, as you still lose it when you die even with this defined.